### PR TITLE
relx 3.5.0

### PR DESCRIPTION
**Pull request by talentdeficit**

### CODE REVIEW

**Comment by dunn** [comment]
Should this be `bin/"relx"`, or did you mean to install `rebar3` as well?
```
==> /usr/local/Cellar/relx/3.5.0/bin/rebar3 --version
Failed to execute: /usr/local/Cellar/relx/3.5.0/bin/rebar3
Error: relx: failed
Failed executing: /usr/local/Cellar/relx/3.5.0/bin/rebar3 --version
```

**Comment by talentdeficit** [comment]
copy/paste error. fixed

File: Library/Formula/relx.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  depends_on "erlang"
+
+  def install
+    system "./rebar3", "update"
+    system "./rebar3", "escriptize"
+    bin.install "_build/default/bin/relx"
+  end
+
+  test do
+    system "#{bin}/relx", "--version"
```
Comment (mikemcquaid) [inline Library/Formula/relx.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. e.g. write out a `relx.config` and do something with it. Thanks1

Comment (DomT4) [inline Library/Formula/relx.rb]: Ping on Mike's comment.

**Comment by DomT4** [comment]
Closing as dead per no response since September 6th. Will take a fresh PR with Mile's comment addressed if you're still interested in getting this in. Thanks for the submission!

**Comment by DomT4** [comment]
Oops. Mike's comment, not Mile's comment. Nobody called Mile commented :smile:.

**Pull request closed without merging**